examples/glamorous-jest-react/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,67 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`enzyme.mount 1`] = `
+.css-1otybxc,
+[data-css-1otybxc] {
+  padding: 4em;
+  background: papayawhip;
+}
+
+.css-1tnuino,
+[data-css-1tnuino] {
+  font-size: 1.5em;
+  text-align: center;
+  color: palevioletred;
+}
+
+<Wrapper>
+  <glamorous(section)>
+    <section
+      className="css-1otybxc"
+    >
+      <Title>
+        <glamorous(h1)>
+          <h1
+            className="css-1tnuino"
+          >
+            Hello World, this is my first glamor styled component!
+          </h1>
+        </glamorous(h1)>
+      </Title>
+    </section>
+  </glamorous(section)>
+</Wrapper>
+`;
+
+exports[`enzyme.render 1`] = `
+.css-1otybxc,
+[data-css-1otybxc] {
+  padding: 4em;
+  background: papayawhip;
+}
+
+.css-1tnuino,
+[data-css-1tnuino] {
+  font-size: 1.5em;
+  text-align: center;
+  color: palevioletred;
+}
+
+<section
+  class="css-1otybxc"
+>
+  <h1
+    class="css-1tnuino"
+  >
+    Hello World, this is my first glamor styled component!
+  </h1>
+</section>
+`;
+
+exports[`enzyme.shallow 1`] = `
+<glamorous(section)>
```

Comment:
I thought the point of shallow render is to not get detailed snapshots. I've usually seen no depth in shallow snapshots. Are you expecting the css to show up as class in this snapshot like render/mount?